src/Components/InstanceTypes/SuggestedInstanceTypesTable.js

```diff
@@ -32,8 +34,27 @@ export default function SuggestedInstanceTypesTable() {
     };
 
     useEffect(() =>{
-        dispatch(loadSuggestedInstanceTypes({ page, perPage }));
-    }, [page, perPage]);
+        dispatch(loadSuggestedInstanceTypes({
+            page, perPage, activeSortDirection, activeSortIndex
+        }));
+    }, [
+        page,
+        perPage,
+        activeSortDirection,
+        activeSortIndex
+    ]);
+
+    const getSortParams = (columnIndex) => ({
+        sortBy: {
+            index: activeSortIndex,
```

Comment:
We can make use of f[indIndex array method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex) here to find the index based on the key if we decide to change the array format.